src/app/public/public.routes.ts

```diff
@@ -36,10 +35,6 @@ export const routes: RoutesWithData = [
         path: 'error',
         component: ItemNotFoundComponent
       },
-      {
-        path: 'search',
```

Comment:
Note that I'm just letting the default error handler take control when someone visits `/p/search`. I'm making the assumption that if we were directly linking to this component we would have, y'know, finished it, but let me know if you think we should redirect to `/p` instead of showing an error!